scania-digital-design-system/sdds
# fix(checkbox): clean up documentation of checkbox

**Describe pull-request**  
Removed native from components path

**Solving issue**  
Fixes: -

**How to test**  
_Add description how to test if possible_
1. Go to storybook link below
2. Check in components -> checkbox
3. Check that the component has no Native in its path name.

**Screenshots**  
-

**Additional context**  
-
